src/main/java/study/racingcar/RacingGame.java

```diff
@@ -0,0 +1,31 @@
+package study.racingcar;
+
+public class RacingGame {
+    private int time;
+    private int[] carPositions;
+    MovableDistance movableDistance;
```

Comment:
인스턴스 변수의 접근 제한자는 private으로 구현합니다.

```suggestion
    private MovableDistance movableDistance;
```